### DISCUSSION TITLE
Silly Achievements? Really?

### DISCUSSION THREAD

**Discussion by RonaldGuilhermePDS** [community/community#19198] [Other Feature Feedback, Questions, & Ideas]
Why implement achievements when the process for receiving them is silly? really?

https://github.com/Schweinepriester/github-profile-achievements

**Comment by D7EAD** [comment]
The question is, why not implement them! Who doesn't love showing off their contributions and achievements?

While they may seem sillly to some, achievements "_[recognize] the many stages of a developer’s coding journey._" Take, for instance, the [Mars 2020 Helicopter Badge](https://github.blog/changelog/2021-04-19-new-badge-for-developers-who-contributed-to-the-mars-2020-helicopter-mission/); this badge not only recognizes an individual developer's contribution(s) to a specific set of important repositories, but a groundbreaking contribution to an off-planet mission!

At the end of the day, achievements are just little badges on your profile; what harm could they do?

**Comment by JhonnyLusonode** [comment]
I believe there are some people who doesn´t care or doesn´t like showing off their achievements but there are some people who like them.

Me personally like them hehe.

For the people who doesn´t like them showing off in their profile they have the option of  hide them on:
 - Settings > Scroll down to "Profile Settings" and tick off the option "Show Achievements on my profile" and update
 
 As shown in picture:
 
![image](https://user-images.githubusercontent.com/43578500/176187882-ef4677b0-dd60-40c0-b36c-a8ee34a9d408.png)

**Comment by GabyCT** [comment]
I really like the achievements, I think they are a motivation specially for contributors to get involve and discover new areas like for example contributing to the Mars Helicopter Badge

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide
